endpoints.yaml

```diff
@@ -0,0 +1,12 @@
+endpoints:
+  - name: companies
+    path: https://pagsegurohml.img.com.br/server/rest/services/Hosted/Roteirizacao/FeatureServer/2/query
+    requests:
+      - name: list
+        vars:
```

Comment:
You are creating the vars, but no using anywhere else. Where do you want to use it? Do you have the curl of the request you are trying to make? This way I can help you better. Since I don't have access to ArcGIS Enterprise.

Reply:
I will see this video